From 35d00dd5a570f7966d4b5fdc423d70781409e192 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 7 May 2018 22:56:09 +0200
Subject: [PATCH 1/1] tools/file2include: avoid incorrect comments

Avoid creating incorrect comments like /* ...*/... */ by printing
'.' instead of '*' inside comments.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 tools/file2include.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/file2include.c b/tools/file2include.c
index e602f937f15..b98af30a728 100644
--- a/tools/file2include.c
+++ b/tools/file2include.c
@@ -84,7 +84,7 @@ int main(int argc, char *argv[])
 			printf("\\x%02x", buf[j]);
 		printf("\"}, /* ");
 		for (j = i; j < i + BLOCK_SIZE && j < count; ++j) {
-			if (buf[j] >= 0x20 && buf[j] <= 0x7e)
+			if (buf[j] != '*' && buf[j] >= 0x20 && buf[j] <= 0x7e)
 				printf("%c", buf[j]);
 			else
 				printf(".");
-- 
2.17.0

